src/mobile/src/main/java/tupperdate/android/utils/ActivityRegistration.kt

```diff
@@ -0,0 +1,17 @@
+package tupperdate.android.utils
+
+import android.graphics.Bitmap
+import androidx.activity.result.ActivityResultLauncher
+
+data class ActivityRegistration(
+    val recipePicture: Registration<Void, Bitmap>,
+)
+
+data class Registration<I, O>(
+    val launcher: ActivityResultLauncher<I>,
+    val data: RegistrationData<O>
+)
+
+data class RegistrationData<O> (
+    var value: O? = null
+)
```

Comment:
I maybe missed something, but do we still use that ?

Reply:
No we don't, I forgot to remove it